Signal/translations/en.lproj/Localizable.strings

```diff
@@ -808,13 +820,20 @@
 /* The subtitle for the messages view title indicates that the title can be tapped to access settings for this conversation. */
 "MESSAGES_VIEW_TITLE_SUBTITLE" = "Tap here for settings";
 
-/* Indicator that separates read from unread messages. */
+/* Indicator that separates read from unread
```

Comment:
Yes, there's a bunch of these mangled messages.  Running `auto-genstrings` fixes them.